src/routes/ProtectedRoute.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { Navigate } from 'react-router-dom';
+
+const ProtectedRoute = ({children}:any) => {
```

Comment:
Hmm, we don't have authentication on the site so we don't need a Protected Route, the option to get to this page will be through "hidden secret egg" as the storybook.